docs/src/whatsnew/latest.rst

```diff
@@ -147,7 +147,7 @@ This document explains the changes made to Iris for this release
    environment info. (:pull:`3990`)
 
 #. `@bjlittle`_ enabled `cirrus-ci`_ compute credits for non-draft pull-requests
-   from collaborators targeting the Iris ``master`` branch. (:pull:`4007`)
+   from collaborators targeting the Iris ``main`` branch. (:pull:`4007`)
```

Comment:
If we include entries like this in What's New (which I approve of), then this PR itself deserves an entry too.

Reply:
Added a whatsnew entry